tests/src/tests/cli.rs

```diff
@@ -153,13 +159,50 @@ async fn disseminate_and_retrieve() {
         .flat_map(|(_, blobs)| blobs)
         .collect();
 
+    // Encode test data
+    let mut bytes: Vec<u8> = config.data.unwrap().into_bytes();
+
+    let remainder = bytes.len() % 31;
+    if remainder != 0 {
+        bytes.resize(bytes.len() + (31 - remainder), 0);
+    }
+    let global_params =
+        kzgrs_backend::global::global_parameters_from_file(&GLOBAL_PARAMS_PATH.to_string())
+            .expect("Global parameters should be loaded from file");
+
+    let params = kzgrs_backend::encoder::DaEncoderParams::new(
+        config.columns.clone(),
+        config.with_cache,
+        global_params,
+    );
+    let da_encoder = kzgrs_backend::encoder::DaEncoder::new(params);
+    let encoded_data = da_encoder.encode(&bytes).unwrap();
+
+    let blob = DaBlob {
+        column: encoded_data.extended_data.columns().next().unwrap(),
+        column_idx: 0,
+        column_commitment: encoded_data.column_commitments[0],
+        aggregated_column_commitment: encoded_data.aggregated_column_commitment,
+        aggregated_column_proof: encoded_data.aggregated_column_proofs[0],
+        rows_commitments: encoded_data.row_commitments.clone(),
+        rows_proofs: encoded_data
+            .rows_proofs
+            .iter()
+            .map(|proofs| proofs.get(0).cloned().unwrap())
+            .collect(),
+    };
+
+    let s_b = Wire::serialize(blob);
+
     // Index zero shouldn't be empty, node 2 replicated both blobs to node 1 because they both
     // are in the same subnetwork.
     for b in node1_idx_0_blobs.iter() {
-        assert!(!b.is_empty())
+        assert!(!b.is_empty());
+        assert_eq!(**b, s_b);
```

Comment:
In case of encoding "hello world" we'll get an encoded data with two columns. Each node will have blobs for both columns, and this assert might be true for only one blob with `column_idx = 0`.

Reply:
Thanks for spotting this. Fixed at https://github.com/logos-co/nomos-node/pull/796/commits/99f342ad00e2e3a8fb2df9855eacbedcf5b71fa3